dotnet/src/Connectors/Connectors.OpenAI/AzureSdk/OpenAIChatMessageContent.cs

```diff
@@ -110,6 +116,7 @@ public IReadOnlyList<OpenAIFunctionToolCall> GetOpenAIFunctionToolCalls()
             }
 
             // Add the additional entry.
+            newDictionary.Add(FunctionToolCallsProperty, toolCalls.OfType<ChatCompletionsFunctionToolCall>().ToList());
             newDictionary.Add(ToolCallsProperty, toolCalls);
```

Comment:
Should we remove this? 

This should only contain the same data as what is stored under `FunctionToolCallsProperty`. We could just serialise the one type of tools we know about and extend this later to serialise other tools.

Reply:
Yeah, I think the only way this goes wrong is if the sdk sends us another kind of tool, which doesn't exist yet. Let's do the simple thing now that doesn't negatively impact the serialized state, and then subsequently follow up with a change to use something that gives us more control over what's serialized and how.

Reply:
Removed the property